### PR TITLE
Turn on format_yard

### LINKED ISSUE #219: Update Annotate Config
## Describe the Feature
**Is your feature request related to a problem? Please describe.**
Annotate offers Yard formatting, and we want to use that to format our documentation.

**Additional context**
We automatically generate documentation using inline documentation and share them at https://rubydoc.info/github/ChaelCodes/HuntersKeepers/Hunter.

## Describe the Technical Implementation of the Solution
**Solution**
https://github.com/ChaelCodes/HuntersKeepers/blob/d611c8e62e4bd64dd94e0f4e650481bcd4915310/lib/tasks/auto_annotate_models.rake#L47-L49

Turn off `format_bare` and turn on `format_yard`, then run `rake db:migrate` to update the annotate docs in each class.

**References**
Any references to materials or help articles that may help in implementing this issue.

### PR DESCRIPTION
**Pull request by JuanVqz**
## Description of Feature or Issue
closes #219

## User-Facing Changes
screenshot_2021-02-11_18:57:15

## Under-the-Hood Changes

Although I don't fully understand why we want to use this syntax type I did it. 😀

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -27,8 +27,8 @@ development:
   <<: *default
   database: hunterskeepers_development
   host: <%= ENV['DB_HOST'] || 'localhost' %>
-  username: <%= ENV['POSTGRES_USER'] || 'root' %>
-  password: <%= ENV['POSTGRES_PASSWORD'] || nil %>
+  username: <%= ENV['POSTGRES_USER'] || "itox" %>
```
Comment (ChaelCodes) [inline config/database.yml:30]: Why did you rename these??

Comment (JuanVqz) [reply config/database.yml]: Sorry, I didn't notice I pushed my Postgres config.

File: config/database.yml [hunk 2]
```diff
@@ -64,8 +64,8 @@ test:
   <<: *default
   database: hunterskeepers_test
   host: <%= ENV['DB_HOST'] || 'localhost' %>
-  username: <%= ENV['POSTGRES_USER'] || 'root' %>
-  password: <%= ENV['POSTGRES_PASSWORD'] || nil %>
+  username: <%= ENV['POSTGRES_USER'] || "itox" %>
+  password: <%= ENV['POSTGRES_PASSWORD'] || "itox" %>
```
Comment (ChaelCodes) [inline config/database.yml:68]: Hey, looks like these are still in here!

**Review by ChaelCodes — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 2 above)
Comment (JuanVqz) [reply config/database.yml]: it's done

**Review by ChaelCodes — APPROVED** [review]
Perfect! Looks good to go now. I can't wait to see how the new attributes look in our yard docs.

**Comment by JuanVqz** [comment]
Where can I see that documentation?

**Comment by ChaelCodes** [comment]
@JuanVqz [Here it is!](https://rubydoc.info/github/ChaelCodes/HuntersKeepers/main/Hunter) It's also linked from the README.
